xaitk_saliency_demo/app/engine/ml_xai.py

```diff
@@ -41,28 +49,28 @@
             "class": SlidingWindowStack,
         },
     },
-    "similarity-saliency": {
-        "_perturb": {
-            "class": SlidingWindow,
-            "params": ["window_size", "stride"],
-        },
+
+    # Similarity
+    "SBSMStack": {
         "_saliency": {
-            "class": SimilarityScoring,
-            "params": ["proximity_metric"],
-        },
+            "class": SBSMStack,
+        }
     },
-    "detection-saliency": {
-        "_perturb": {
-            "class": RISEGrid,
-            "params": ["n", "s", "p1", "seed", "threads"],
-        },
+
+    # Object Detection
+    "DRISEStack": {
         "_saliency": {
-            "class": DRISEScoring,
-            "params": ["proximity_metric"],
-        },
+            "class": DRISEStack
+        }
+    },
+    "RandomGridStack": {
+        "_saliency": {
+            "class": RandomGridStack
+        }
     },
```

Comment:
You seems to remove all the "params"? Are those method not expecting any configuration parameters?